### PR TITLE
Annotate Grafana as optional in architecture docs

### PR DESCRIPTION
**Pull request by nealormsbee**

### CODE REVIEW

**Comment by dwbrown2** [comment]
Looks good, what do you think about marking optional here too:

https://guide.kubecost.com/hc/en-us/articles/4407595941015-Custom-Prometheus#custom-grafana
https://guide.kubecost.com/hc/en-us/articles/4423256582551-Kubecost-Secondary-Clusters
https://guide.kubecost.com/hc/en-us/articles/4407601795863-Installing-in-Air-gapped-Environments

There's a chance there are other places... just found these with a quick search.

**Comment by nealormsbee** [comment]
I'll make a pass on the other documents that mention Grafana.

Somewhat related -- I know we have release notes. For larger / significant changes like making Grafana default to disabled, do we have a good channel for broadcasting those changes to users ahead of time?

**Comment by nealormsbee** [comment]
A search through the docs turned up 14 pages referencing Grafana, but IMO only 

https://guide.kubecost.com/hc/en-us/articles/4407601795863-Installing-in-Air-gapped-Environments
and
https://guide.kubecost.com/hc/en-us/articles/4423256582551-Kubecost-Secondary-Clusters

need updating (in addition to the architecture doc). The one about custom Grafana is all in the context of wanting to connect to an already-existing Grafana deployment.

**Comment by dwbrown2** [comment]
Can you pick for another review right around release time? I don't think we want to merge until then.

**Review by mbolt35 — APPROVED** [review]
LGTM

File: custom-prom.md [hunk 1]
```diff
@@ -131,9 +132,11 @@ grafana:
 
 For Option 2, ensure that the following flags are set in your Operator deployment:
 
-1. sidecar.dashboards.enabled = true  
+1. sidecar.dashboards.enabled = true
 2. sidecar.dashboards.searchNamespace isn't restrictive, use `ALL` if Kubecost runs in another namespace.
 
+Note that with Option 2, the Kubecost UI cannot link to the Grafana dashboards unless `kubecostProductConfigs.grafanaURL` is set, either via the Helm chart, or via the Settings page as described in Option 1.
+
```
Comment (nealormsbee) [inline custom-prom.md:139]: Relevant change. The rest is Prettier cleanup

File: air-gapped.md [hunk 2]
```diff
@@ -23,10 +23,12 @@ for prod-x.xx.x. [Latest releases can be found here](https://github.com/kubecost
 - prom/prometheus:v2.31.1
 - prom/node-exporter:v0.18.1
 - prom/alertmanager:v0.23.0
-- grafana/grafana:8.3.2
 - jimmidyson/configmap-reload:v0.7.1
 - kiwigrid/k8s-sidecar:1.15.4 (can be optional if not using Grafana sidecar)
 
+### Prometheus - Optional
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply air-gapped.md:29]: I think that works!

**Comment by nealormsbee** [comment]
Resolved conflicts -- merging to mark Grafana as optional in 1.93

**Pull request merged**